lib/api/unittest/CDataFrameAnalyzerTrainingTest.cc

```diff
@@ -1523,6 +1519,255 @@ BOOST_AUTO_TEST_CASE(testClassificationIncrementalTraining) {
             row->copyTo(newTrainingData.back().begin());
         }
     });
+
+    frame->resizeRows(0);
```

Comment:
This seems to complete this test but it's not strictly related to the field order stuff, right?

Reply:
This is just the diff getting confused because there is similarity to test I added. If you scroll down you'll see that this was essentially as it was before.